dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/validation/validator/event/DataRelationsValidator.java

```diff
@@ -322,4 +332,28 @@ private Program getEnrollmentProgramFromEvent(TrackerBundle bundle, Event event)
     }
     return null;
   }
+
+  /**
+   * Given a program with registration, check whether an event in the preheat or payload refers to
+   * enrollment with an existing tracked entity. This should not happen because an enrollment
```

Comment:
```suggestion
   * enrollment with an existing tracked entity. This check should not fail because an enrollment
```

is clearer I think

Reply:
I've added another comment as suggested by Marc in the master review